pkg/epinio/pages/c/_cluster/applications/createapp/index.vue

```diff
@@ -111,6 +111,10 @@ export default Vue.extend<Data, any, any, any>({
         // app chart actually belongs in config, so stick it in there
         this.value.configuration = this.value.configuration || {};
         this.set(this.value.configuration, { appchart: appChart });
+        // FIX: this is a hack to get the app chart to show up in the UI
```

Comment:
I'm not sure this is right, appChart is from memory the Application Template and can be set even with github is chose.

Calling `this.set(this.value.configuration, { environment: { appDeployment` will wipe out anything previously in conviguration, and also remove anything previously set in `environment`